src/main/java/org/folio/des/scheduling/quartz/OldJobDeleteScheduler.java

```diff
@@ -0,0 +1,76 @@
+package org.folio.des.scheduling.quartz;
+
+import static org.folio.des.scheduling.quartz.QuartzConstants.TENANT_ID_PARAM;
+
+import java.util.Set;
+import java.util.TimeZone;
+import java.util.UUID;
+
+import org.folio.des.scheduling.quartz.job.bursar.OldDeleteJob;
+import org.quartz.CalendarIntervalScheduleBuilder;
+import org.quartz.JobBuilder;
+import org.quartz.JobDetail;
+import org.quartz.JobKey;
+import org.quartz.Scheduler;
+import org.quartz.Trigger;
+import org.quartz.TriggerBuilder;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import lombok.extern.log4j.Log4j2;
+
+@Service
+@Log4j2
+public class OldJobDeleteScheduler {
+  private final Scheduler scheduler;
+  private final String timeZone;
+
+  public OldJobDeleteScheduler(Scheduler scheduler, @Value("${folio.quartz.bursar.timeZone}") String timeZone) {
+    this.scheduler = scheduler;
+    this.timeZone = timeZone;
+  }
+
+  @Transactional
+  public void scheduleOldJobDeletion(String tenantId) {
+    JobKey jobKey = getDeleteJobKey(tenantId);
+    JobDetail jobDetail = getDeleteJobDetail(tenantId, jobKey);
+    try {
+      if (scheduler.checkExists(jobKey)) {
+        scheduler.deleteJob(jobKey);
+        scheduler.scheduleJob(jobDetail, Set.of(getDeleteJobTrigger(tenantId)), false);
+      } else {
+        scheduler.scheduleJob(jobDetail, Set.of(getDeleteJobTrigger(tenantId)), false);
```

Comment:
There's a simpler api for scheduling job with single trigger: https://www.quartz-scheduler.org/api/2.1.7/org/quartz/Scheduler.html#scheduleJob(org.quartz.JobDetail,%20org.quartz.Trigger)